### PR TITLE
feature: update HF imageUris for PyTorch 1.8.1

### PR DESCRIPTION
**Pull request by ahsan-z-khan**
*Issue #, if available:*

*Description of changes:*
Update ImageUri to include PyTorch 1.8 images to HF
*Testing done:*

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

#### General

- [x] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [x] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md#committing-your-change)
- [ ] I have passed the region in to all S3 and STS clients that I've initialized as part of this change.
- [ ] I have updated any necessary documentation, including [READMEs](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) and [API docs](https://github.com/aws/sagemaker-python-sdk/tree/master/doc) (if appropriate)

#### Tests

- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have checked that my tests are not configured for a specific region or account (if appropriate)
- [ ] I have used [`unique_name_from_base`](https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/utils.py#L77) to create resource names in integ tests (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/sagemaker/image_uri_config/huggingface.json [hunk 1]
```diff
@@ -211,6 +212,36 @@
                     },
                     "repository": "huggingface-pytorch-training"
                 },
+                "pytorch1.8.1": {
+                    "py_versions": ["py36"],
+                    "registries": {
+                        "af-south-1": "626614931356",
+                        "ap-east-1": "871362719292",
+                        "ap-northeast-1": "763104351884",
+                        "ap-northeast-2": "763104351884",
```
Comment (saimidu) [inline src/sagemaker/image_uri_config/huggingface.json:221]: Please also add `ap-northeast-3`. There should be 25 regions in total.

**Review by saimidu — COMMENTED** [review]
LGTM. Just adding the `ap-northeast-3` region to the training image region list would make this PR good to go.

**Review by shreyapandit — APPROVED** [review]

**Review by philschmid — APPROVED** [review]
LGTM! thanks for adding already inference to it.

**Comment by philschmid** [comment]
Hey @ahsan-z-khan , 
We are currently releasing new Images. You can take a look here:https://github.com/aws/deep-learning-containers/issues/1326#issuecomment-920562320 
Maybe can add the config in this PR here as well and the merge after the new PyTorch Container are available

**Comment by ahsan-z-khan** [comment]
@philschmid , Are those update for 4.10 transformer?

**Comment by philschmid** [comment]
> @philschmid , Are those update for 4.10 transformer?

Yes ! new Container for Pytorch 1.8. for inference and training with Transformers 4.10 same for Tensorflow 2.4

**Comment by philschmid** [comment]
Images are released when can we get this out?

**Comment by ahsan-z-khan** [comment]
> Images are released when can we get this out?

@philschmid , this is dependent on #2642 

We are updating the SDK to support cuda111 and later.

**Comment by philschmid** [comment]
any ETA on this?

**Comment by philschmid** [comment]
FYI we also merged new Images for PyTorch 1.9 and TF2.5 
https://github.com/aws/deep-learning-containers/pull/1354
This need to be added soon as well

**Comment by ahsan-z-khan** [comment]
@philschmid , trying to merge today.

**Comment by ahsan-z-khan** [comment]
closing in regards of #2642

**Pull request closed without merging**